### PR TITLE
[14.0] [MIG] ebill_paynet_customer_free_ref: Migrate to version 14.0

### PR DESCRIPTION
**Pull request by kv1612**
Depends on 
- [x] https://github.com/OCA/l10n-switzerland/pull/595
- [x] https://github.com/OCA/edi/pull/452
- [x] https://github.com/OCA/edi/pull/299

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+vcrpy
+xmlunittest
+odoo14-addon-ebill-paynet @git+https://github.com/OCA/l10n-switzerland@refs/pull/595/head#subdirectory=setup/ebill_paynet
```
Comment (simahawk) [inline test-requirements.txt]: Base PR is getting merged, remember to drop these

**Review by simahawk — COMMENTED** [review]
LG

File: test-requirements.txt (hunk 1 above)
Comment (simahawk) [reply test-requirements.txt]: Please, next time remember to link depending PRs in the description (I added them now).

**Review by sebalix — APPROVED** [review]

**Pull request merged**